ansible_mitogen/strategy.py

```diff
@@ -48,6 +48,8 @@
 import ansible.template
 import ansible.utils.sentinel
 
+from ansible.plugins.loader import get_with_context_result
```

Comment:
Personal preference, use absolute imports when possible. It makes it easier to distinguish what belongs to `ansibler.*` and what belongs to `ansible_mitogen.*`, or `mitogen.*`

Reply:
Gotcha - I had some issues getting an absolute import working in this file due to other imports masking things. Tried a few more ways just now but didn't get anywhere.

Reply:
I tried some variations in https://github.com/moreati/mitogen/commit/25da0f71e6c3c2f33071dc8915d6781fc78f4465. We could eliminate this import entirely by using

```python
result = ansible_mitogen.loaders.action_loader__get_with_context(...)
...
return result._replace(object=adorned_klass)
```

It loses a little clarity. I'm in favour of the tradeoff, but ready to be convinced otherwise.

Reply:
I'm not very keen on this alternative for a few reasons - clarity, as you mentioned, but also the fact that we're now relying on `_replace` which is a protected attribute. Adding to that, we've now also leaked the name of the tuple object into our code; meaning that if Ansible ever goes and changes it, our code is broken and needs an update.

Reply:
> I'm not very keen on this alternative

On reflection I agree. I'd still like to use an absolute import. I'll investigate.

Reply:
Absolute import seems to have worked in https://github.com/mitogen-hq/mitogen/pull/1220/commits/8273eac1d650033bd8f36115617b25e3aad1cdb9